### PR TITLE
[MIG][14.0]sale_order_line_description

### PR DESCRIPTION
**Pull request by SimoneVagile**

### CODE REVIEW

File: sale_quotation_number/__manifest__.py~HEAD [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2010-2012 Andy Lu <andy.lu@elico-corp.com> (Elico Corp)
```
Comment (emagdalenaC2i) [inline sale_quotation_number/__manifest__.py~HEAD]: I think you should delete this file

File: sale_quotation_number/i18n/sale_quotation_number.pot~HEAD [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# Translation of Odoo Server.
```
Comment (emagdalenaC2i) [inline sale_quotation_number/i18n/sale_quotation_number.pot~HEAD]: I think you should delete this file

**Review by emagdalenaC2i — APPROVED** [review]
LGTM just a minor change deleting 2 extra files

**Review by dsolanki-initos — APPROVED** [review]
Functional and code review. LGTM

**Review by danimv5 — APPROVED** [review]
Functional review!! LGTM

**Comment by dsolanki-initos** [comment]
It seems that PR is ready to merge can someone please merge this PR.

**Comment by dsolanki-initos** [comment]
@OCA/crm-sales-marketing-maintainers can this be merged?

**Comment by dsolanki-initos** [comment]
Can this PR be merged?

**Comment by HaraldPanten** [comment]
No, it can't.

You should consider @emagdalenaC2i comments.

**Comment by SimoneVagile** [comment]
@emagdalenaC2i files deleted

**Comment by dsolanki-initos** [comment]
@SimoneVagile can you please have look at the above comments and resolve it so that we can merge this PR ASAP.

**Pull request merged**